weather_forecast_retrieval/hrrr_preprocessor.py

```diff
@@ -13,6 +13,17 @@ class HRRRPreprocessor():
     FMT1 = '%Y%m%d'
     FMT2 = '%H'
 
+    VARIABLES = [
+        'TMP:2 m',
+        'RH:2 m',
+        'UGRD:10 m',
+        'VGRD:10 m',
+        'APCP:surface',
+        'DSWRF:surface',
+        'HGT:surface',
+        'TCDC:entire atmosphere'
+    ]
```

Comment:
`MappingProxyType` ?

Reply:
`MappingProxyType` only works with dicts, a FrozenSet would probably fit the bill here. The main problem here is that the test files are missing `TCDC` so without adding a bunch of work or regenerating test files, this should work for now. It's not as complicated as SMRF so hopefully that will be okay.